src/ng-select/lib/ng-select.component.ts

```diff
@@ -116,6 +117,7 @@ export class NgSelectComponent implements OnDestroy, OnChanges, OnInit, AfterVie
 	@Input() labelForId = null;
 	@Input() inputAttrs: { [key: string]: string } = {};
 	@Input({ transform: numberAttribute }) tabIndex: number;
+	tabFocusOnClearButton = input(true, { transform: booleanAttribute });
```

Comment:
I think the default value should be either false or if possible the same value as the config value. Otherwise, the config value being set to false will not override this unless you also specify it on every control as well.